Sources/Conformance/failure_list_swift.txt

```diff
@@ -1 +1,2 @@
-# No known failures
+Required.Proto3.JsonInput.DurationNegativeNanos.JsonOutput
+Required.Proto3.JsonInput.DurationNegativeNanos.ProtobufOutput
```

Comment:
@thomasvl @tbkka here I am following this recent change on google/protobuf where some tests were added to the failure list: https://github.com/protocolbuffers/protobuf/commit/4ea4d1d91a65861a16f4bbb231f1bf2daf795d8a

Let me know if I shouldn't do it this way.

Reply:
If this is a new test, please spin out a different PR and also open an issue for the tests failing.

Reply:
I got a few minutes, so I've opened #1257

Reply:
Awesome! Thank you very much @thomasvl.